### ISSUE TITLE
[CoE Starter Kit - Feature]: CoE utility to update Owner/Derived Owner for orphaned SharePointFormApps

### ISSUE THREAD

**Issue by jmhack01** [microsoft/coe-starter-kit#8408]
### Is your feature request related to a problem? Please describe.

I am trying to clean-up orphaned SharePointFormApps, but there is no way to do it natively in the Power Platform administration or COE toolkit.  All SharePoint form apps are managed through the Sharepoint interface. While another user can modify the app from SharePoint, the owner doesn't change.  This creates an issue with the owner has left the organization and the SharePointFormApp is orphaned.

### Describe the solution you'd like

A CoE toolkit utility that will let you update the Owner for a SharePointFormApp.  The Power Platform flows utility has a similar feature to update a "derived owner", so at least the CoE toolkit reporting recognized a new owner.  

### Describe alternatives you've considered

Creating a custom flow that modifies the PowerApps App table to change the owner

### Additional context?

I have noticed other users on the internet with similar issue. Here is a Reddit article describing such: https://www.reddit.com/r/PowerApps/comments/wwt2mb/change_owner_of_sharepointformapp_pa_in_spo_not/

AB#3462

**Comment by Jenefer-Monroe** [comment]
Thank you for the proposal. Putting on the backlog for PM consideration.

**Comment by Alyson95170** [comment]
Hi, I am also cleaning up the orphaned objects and facing the same issue, will this be considered for a future release?

**Comment by Jenefer-Monroe** [comment]
Hello unfortunately we have some large competing priorities at the moment and so this is not something we will be funding soon.

**Comment by Alyson95170** [comment]
Thanks Jenefer, is there a way to stop the manager of the former employee receiving the notifications each week for the orphaned objects?

**Comment by Jenefer-Monroe** [comment]
Unfortunately no, not other than having them assign the new owners using the utilities provided. 
We do not have an "ignore" feature for the orphan feature area. 
If they assign the new owners though, then they will not get the message any longer.

**Comment by Jenefer-Monroe** [comment]
No, unfortunately we do not have this item funded today. We are heads down in funding the new Data Export feature area.

**Comment by Service-WebDevPowerP** [comment]
Does anyone know if there is a PS script that can be run to update the owner?

**Comment by AmarSaiRam** [comment]
We are consolidating multiple related enhancement requests into a single comprehensive issue to streamline tracking and implementation. This ticket is now closed because its scope is fully covered under the new consolidated issue focused on centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: FEATURE TITLE Enhancement: Centralized Management for Orphaned Compone…](https://github.com/microsoft/coe-starter-kit/issues/10319)

Thank you for your valuable input and collaboration
